src/private/conversations.ts

```diff
@@ -0,0 +1,66 @@
+import { ensureInitialized, sendMessageRequest } from "../internal/internalAPIs";
```

Comment:
export all these function to public by adding _export * from "./conversation"; inside ./src/private/index.ts file.

Reply:
we need to this for all new file added inside private or public folder.